x-pack/metricbeat/module/aws/utils.go

```diff
@@ -45,6 +45,10 @@ func GetListMetricsOutput(namespace string, regionName string, svcCloudwatch clo
 
 	listMetricsInput := &cloudwatch.ListMetricsInput{
 		NextToken: nextToken,
+		// To filter the results to show only metrics that have had data points published
+		// in the past three hours, specify this parameter with a value of PT3H.
+		// Please see https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_ListMetrics.html for more details.
+		RecentlyActive: "PT3H",
```

Comment:
nit: there is a constant for this value in the cloudwatch types package

Reply:
good call thanks!